Samples/PublicSamples/RecordingBot/src/RecordingBot.Services/Authentication/UserPasswordAuthenticationProvider.cs

```diff
@@ -1,21 +1,3 @@
-// ***********************************************************************
-// Assembly         : RecordingBot.Services
-// Author           : JasonTheDeveloper
-// Created          : 09-07-2020
-//
-// Last Modified By : dannygar
-// Last Modified On : 08-17-2020
-// ***********************************************************************
-// <copyright file="UserPasswordAuthenticationProvider.cs" company="Microsoft">
-//     Copyright ©  2020
-// </copyright>
-// <summary></summary>
-// ***********************************************************************>
-
-// THIS CODE HAS NOT BEEN TESTED RIGOROUSLY.USING THIS CODE IN PRODUCTION ENVIRONMENT IS STRICTLY NOT RECOMMENDED.
```

Comment:
Not a Ghost comment

Reply:
True, but we have tested this rigorously by now